bitnami/grafana-loki/templates/alloy/secret.yaml

```diff
@@ -0,0 +1,116 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+
+{{- if and .Values.grafanaalloy.enabled .Values.grafanaalloy.alloy.createSecret }}
```

Comment:
Why is the 2nd condition required?

Reply:
I want to offer the possibility to use an existingSecret that the user has created so:
1. Using existingSecret makes Alloy to avoid its creation
2. Using createSecret (maybe I can change this name), I ask loki templates to creates a secret for Alloy configurati

Reply:
We decided to keep it there